man/powerman.1.in

```diff
@@ -46,44 +33,72 @@ ranges.
 .I "-Q, --query targets"
 Query plug status of specific targets.
 .TP
-.I "-b, --beacon-all"
-Query beacon status of all targets (if implemented by RPC).
+.I "-h, --server-host host[:port]"
+Connect to a powerman daemon on non-default host and optionally port.
 .TP
-.I "-B, --beacon targets"
-Query beacon status of specific targets (if implemented by RPC).
+.I "-x, --exprange"
+Expand host ranges in query responses.
 .TP
-.I "-t, --temp-all"
-Query node temperature of all targets (if implemented by RPC).
-Temperature information is not interpreted by powerman and is reported
-as received from the RPC on one line per target, prefixed by target name.
+.I "-l, --list"
+List available targets.  If possible, output will be compressed into
+a host range (see TARGET SPECIFICATION below).
 .TP
-.I "-P, --temp targets"
-Query node temperature of specific targets (if implemented by RPC).
+.I "-V, --version"
+Display the powerman version number and exit.
 .TP
 .I "-L, --license"
 Show powerman license information.
 .TP
-.I "-h, --server-host host[:port]"
-Connect to a powerman daemon on non-default host and optionally port.
+.I "-g, --genders"
+Interpret targets as genders attributes rather than node names.  Each attribute
+is expanded to the list of nodes that have that attribute, then those lists
+are combined to make a list of target node names.
```

Comment:
should wrap this around the appropriate guards if powerman built with/without genders?

Reply:
powerman will give you a nice error if you try to use that when it's not configured
```
$ powerman -g -1 foo
powerman: not configured with genders support
```
so I figured it's not necessary to also spell that out in the man page.